Maths/fibonacci2.py

```diff
@@ -0,0 +1,26 @@
+# Fibonacci Sequence Using Recursion
+
+
+def fibonacci(limit):
+    first_number = 1
```

Comment:
Make it start from **0**.
[Why does the Fibonacci Series start with 0, 1?](https://math.stackexchange.com/questions/158699/why-does-the-fibonacci-series-start-with-0-1)